modules/oadp-release-notes-1-3-4.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-4_{context}"]
+= {oadp-short} 1.3.4 release notes
+
+The {oadp-first} 1.3.4 release notes list resolved issues and known issues.
+
+[id="resolved-issues-1-3-4_{context}"]
+== Resolved issues
+
+.The backup spec.resourcepolicy.kind parameter is now case insensitive
+
+Previously, the `backup spec.resourcepolicy.kind` parameter was only supported with a lower-level string. With this fix, it is now case insensitive. link:https://issues.redhat.com/browse/OADP-2944[{oadp-short}-2944]
+
+.olm.maxOpenShiftVersion prevents cluster upgrade to OCP 4.16 version
+
+The cluster `operator-lifecycle-manager` operator must not be upgraded between minor {OCP} versions. Use the `olm.maxOpenShiftVersion` parameter to prevent upgrading to {OCP} 4.16 version when {oadp-short} 1.3 is installed. 
+To upgrade to {OCP} 4.16 version, upgrade {oadp-short} 1.3 on OCP 4.15 version to {oadp-short} 1.4. link:https://issues.redhat.com/browse/OADP-4803[{oadp-short}-4803]
+
+.BSL and VSL are removed from the cluster
+
+Previously, when any Data Protection Application (DPA) was modified to remove the Backup Storage Locations (BSL) or Volume Snapshot Locations (VSL) from the `backupLocations` or `snapshotLocations` section, BSL or VSL were not removed from the cluster until the DPA was deleted.
+With this update, BSL/VSL are removed from the cluster. link:https://issues.redhat.com/browse/OADP-3050[{oadp-short}-3050]
+
+.DPA reconciles and validates the secret key
+
+Previously, the Data Protection Application (DPA) reconciled successfully on the wrong Volume Snapshot Locations (VSL) secret key name.
+With this update, DPA validates the secret key name before reconciling on any VSL. link:https://issues.redhat.com/browse/OADP-3052[{oadp-short}-3052]
+
+
+.Velero's cloud credential permissions are now restrictive
+
+Previously, Velero's cloud credential permissions were mounted with the 0644 permissions. As a consequence, any one could read the `/credentials/cloud` file apart from the owner and group making it easier to access sensible information such as storage access keys.
+With this update, the permissions of this file are updated to 0640 and the file cannot be accessed by other users.
+
+
+.Backup completes without deleting the Volumesnapshot resource 
+
+Previously, the `Volumesnapshot` resource was deleted by the `argocd` tool and the backup failed due to this. 
+With this fix, `Volumesnapshot` is not deleted until the backup is successful and {oadp-short} has reconciled all resources properly. link:https://issues.redhat.com/browse/OADP-4736[{oadp-short}-4736]
```

Comment:
we have not changed the behavior of Velero, we only added a warning that argo and velero are managing the same namespace.  See the PR https://github.com/openshift/velero/pull/354

Reply:
@weshayutin, I prepared this content based on the [PR](https://github.com/vmware-tanzu/velero/issues/7905). Could you please let me know if the added doc-text needs any modification?

Reply:
@apurvabhide17 Adding to what @weshayutin is saying, we are not fixing anything here. We only worked on adding a warning for users. The [change](https://github.com/openshift/velero/pull/354) adds a warning to velero backup operation if there are namespaces included in the backup which are managed by ArgoCD.

Reply:
@shubham-pampattiwar @weshayutin, I've updated the doc-text as follows -
```
.Warning is displayed when ArgoCD managed namespace is included in the backup

A warning is displayed during the backup operation when ArgoCD and Velero manage the same namespace. 
```
Is this correct?